pkg/cedar/intake/models/easi_intake.go

```diff
@@ -1,53 +1,61 @@
 package models
 
 // NOTE: this type is used to create a schema used by the CEDAR Intake API
-// When changing this type, update the version for it in pkg/cedar/intake/translation/constants.go (IntakeInputSchemaEASIIntakeVersion)
+// When changing this type, add a new version for it in pkg/cedar/intake/translation/constants.go
```

Comment:
This change (just the comment) undoes work from https://github.com/CMSgov/easi-app/pull/1668; please manually reverse it.